dashboard/app/models/script_level.rb

```diff
@@ -575,9 +572,11 @@ def summarize_for_teacher_panel(student, teacher = nil)
       id: level.id.to_s,
       contained: contained,
       submitLevel: level.properties['submittable'] == 'true',
-      paired: paired,
+      paired: is_driver || is_navigator || false,
```

Comment:
Could this just be `is_driver || is_navigator`?

Reply:
I wanted this to be a boolean (and not nil) when user_level is nil above.  I could change this to `!!(is_driver || is_navigator)` if that's clearer?

Reply:
Oh, got it! No I think this is fine, I just didn't think of that. It would be great to cover this with a unit test to make sure someone doesn't accidentally refactor it to `is_driver || is_navigator` (like me in the future)